src/main/java/com/github/pedrovgs/androidwifiadb/model/Device.java

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-package com.github.pedrovgs.androidwifiadb;
+package com.github.pedrovgs.androidwifiadb.model;
 
 public class Device {
 
-  private final String name;
+  private String name;
```

Comment:
If the device is not authorized, we get the device name in not human readable format - "30e003beb63 unathorized usb:...". 
Once the user authorises device, we should update device name.